### ISSUE TITLE
HAOS: snmpd[139]: systemstats_linux: unexpected header length in /proc/net/snmp. 237 != 224

### ISSUE THREAD

**Issue by pvince** [mib1185/ha-addon-snmpd#15]
I am running Home Assistant OS with the following versions
```
Core 2024.11.2
Supervisor 2024.11.3
Operating System 13.2
Frontend 20241106.2
```

I have the SNMP addon installed and the SNMP data is being aggregated in LibreNMS.

I have had to directly access HAOS directly and interact with Home Assistant via the HA CLI, and when I do so I noticed that the following error message is repeatedly blasted directly to the terminal:
```
snmpd[139]: systemstats_linux: unexpected header length in /proc/net/snmp. 237 != 224

snmpd[139]: systemstats_linux: unexpected header length in /proc/net/snmp. 237 != 224

snmpd[139]: systemstats_linux: unexpected header length in /proc/net/snmp. 237 != 224

snmpd[139]: systemstats_linux: unexpected header length in /proc/net/snmp. 237 != 224
```

This message is output periodically and makes it hard to work with the command line.

I did some googling and found that this is a known & fixed bug with net-snmp:
* https://github.com/net-snmp/net-snmp/issues/829
* https://github.com/net-snmp/net-snmp/pull/785

Since HAOS has a read-only filesystem, there is no easy way for me to fix this with workarounds that are outlined elsewhere:
* https://forum.proxmox.com/threads/snmpd-2644-systemstats_linux-unexpected-header-length-in-proc-net-snmp-237-224.146957/#post-677327

**Comment by mib1185** [comment]
This has already been reported in https://github.com/mib1185/ha-addon-snmpd/issues/12#issuecomment-2276269147. We need to wait for the next release of net-snmp

**Comment by snix47** [comment]
6 months has passed, and the problem is the same for me, anyone else still having the same problem?
snmpd[139]: systemstats_linux: unexpected header length in /proc/net/snmp. 237 != 224

//Rille

**Comment by mib1185** [comment]
we're still waiting for a new release of net-snmp which contains the fix https://github.com/net-snmp/net-snmp/pull/785

Therefore there is a pinned issue report which also mention this 😉

**Comment by snix47** [comment]
> we're still waiting for a new release of net-snmp which contains the fix [net-snmp/net-snmp#785](https://github.com/net-snmp/net-snmp/pull/785)
> 
> Therefore there is a pinned issue report which also mention this 😉

Thank You so much for that explanation, now it´s clear even to me, thanks'.
